src/Navigation/Extra.elm

```diff
@@ -0,0 +1,26 @@
+module Navigation.Extra exposing (locationFromString)
+
+{-| TODO: this module should implement the algorithm described at
+<https://www.w3.org/TR/2012/WD-url-20120524/>
+-}
+
+import Navigation
+
+
+{-| Returns `Nothing` if the given string is not a valid absolute URL.
```

Comment:
Might be good to add an example of an absolute URL here

Reply:
Fixed in https://github.com/avh4/elm-program-test/pull/1/commits/d429b61182e83a43d114111f0830c311dce6d003